galaxy_ng/app/auth/auth.py

```diff
@@ -65,58 +60,15 @@ def authenticate(self, request):
             last_name=last_name
         )
 
-        self._ensure_synclists(group)
-
         return user, {'rh_identity': header}
 
     def _ensure_group(self, account_scope, account):
-        """Create a auto group for the account and create a synclist distribution"""
+        """Create a auto group for the account"""
 
         with transaction.atomic():
             group, created = Group.objects.get_or_create_identity(account_scope, account)
         return group, created
 
-    def _ensure_synclists(self, group):
-        with transaction.atomic():
-            # check for existing synclists
-            perms = ['galaxy.view_synclist']
-
-            synclists_owned_by_group = \
-                shortcuts.get_objects_for_group(group, perms, klass=SyncList,
-                                                any_perm=False, accept_global_perms=True)
```

Comment:
The SyncList object is still being used on CRC. We are only removing the SyncList.repository and the curation of those repos.